.github/workflows/build.yml

```diff
@@ -0,0 +1,71 @@
+name: Build hjson-cli Releases
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+**'
+
+jobs:
+  
+  build_hjson-cli_releases:
+    runs-on: ${{ matrix.os }}
+    
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+          
+      - name: Build the ${{ runner.os }} hjson-cli binary
+        run: |
+          mkdir binaries
+          cd hjson-cli
+          go build
+          
+      - name: Rename hjson-cli binary to reflect ${{ runner.os }}
+        run: |
+          mv hjson-cli/hjson-cli.exe binaries/hjson-cli_${{ github.ref_name }}_${{ runner.os }}.exe
+        if: ${{ contains(matrix.os, 'windows') }}
+        
+      - name: Rename hjson-cli binary to reflect ${{ runner.os }}
+        run: |
+          mv hjson-cli/hjson-cli binaries/hjson-cli_${{ github.ref_name }}_${{ runner.os }}
+        if: runner.os != 'Windows'
+        
+      - name: Upload hjson-cli artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: output
+          path: binaries/*
+          if-no-files-found: error
+  
+  
+  release_artifacts:
+    needs: [build_hjson-cli_releases]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download actifacts
+      uses: actions/download-artifact@v2
+      with:
+        path: artifacts
+        
+    - name: Show the downloaded artifacts
+      run: |
+        pwd
+        ls -laR *
+        
+    - name: Release binaries
+      uses: ncipollo/release-action@v1
+      with:
+        # ncipollo/release-action needs a tag! Either a usual "GIT TAG" or an dedicated TAG, see below!
+        # set a TAG if you want to build a release, i.e. via "workflow_dispatch" on GitHub _AND_ do not push a regular GIT TAG
+        # and the other way around, if you want to build releases based on pushed GIT TAGs, make sure you un-comment the "tag:" line below!
+        tag: ${{ github.ref_name }} 
+        allowUpdates: true

```

Comment:
Releases are only build when a TAG is pushed... and only based on this pushed TAG!
`allowUpdates` is very useful if you had a TAG (and consequently a Release based on it) and then for whatever reason delete and create the same TAG again...

From my intention, namely to provide "official" releases based on the released TAGs, `draft' releases make little sense in this context.
BUT if you want to have `draft` releases, a changed context in `on.push` would IMHO make more sense... depending WHEN you'd like to have `drafts`...

Reply:
In my opinion a release should never change after it has been made official. The draft release would allow a human to review the release (and edit the text) before pushing the button to change the draft into an official release. The drafts are only visible for team members, it's not like pre-releases.

Reply:
ok, will change the YML file and sent another commit soon then ;)